requirements-dev.txt

```
@@ -1,4 +1,4 @@
-pytest==3.10.1
+#pytest==3.10.1
```

I think this should be:
```suggestion
pytest
```